src/main/java/com/redhat/cloud/policies/app/health/ScheduledStatusProducer.java

```diff
@@ -44,11 +47,14 @@
     @RestClient
     PolicyEngine engine;
 
-    //  // Quarkus only activates this after the first REST-call to any method in this class
-    @Gauge(name = "status_isDegraded", unit = MetricUnits.NONE, absolute = true,
-            description = "Returns 0 if good, value > 0 for number of entries in the status message")
-    int isDegraded() {
-        return StuffHolder.getInstance().getStatusInfo().size();
+    @Inject
+    MeterRegistry registry;
+
+    Gauge degraded;
```

Comment:
Do we need a class field for that?

Wouldn't this work?
```
    @PostConstruct
    void init() {
        Gauge.builder("status_isDegraded", () -> StuffHolder.getInstance().getStatusInfo().size()).register(registry);
    }
```

Reply:
it would. changed.